pkg/lib/cockpit-components-table.scss

```diff
@@ -1,14 +1,90 @@
 @import "~@patternfly/patternfly/components/Table/table.scss";
 @import "~@patternfly/patternfly/components/Table/table-grid.scss";
 
-// Display the actions next to the header inline
-header {
-    display: flex;
-    > :only-child {
+.ct-table {
+    &.pf-m-compact {
+        > thead, > tbody {
+            > tr:not(.pf-c-table__expandable-row) {
+                // We actually want the normal font size for our lists
+                --pf-c-table-cell--FontSize: var(--pf-global--FontSize--md);
```

Comment:
PF has smaller font for compact lists.
I think this makes sense or if not it should be discussed also with PF team.
 
In this case:
--pf-c-table-cell--FontSize: var(--pf-c-table--m-compact--FontSize);
--pf-c-table--m-compact--FontSize: var(--pf-global--FontSize--sm);

Reply:
We probably should remove `pf-m-compact` from our JSX/HTML and redo the padding ourselves. That's what we really want from the pf-m-compact in this way. Then we wouldn't have to change the font back... just the padding. We're already having to do alignment and padding changes anyway, so... it probably makes sense?